internal/controller/drplacementcontrol.go

```diff
@@ -918,6 +917,10 @@ func (d *DRPCInstance) ensureFailoverActionCompleted(srcCluster string) (bool, e
 	return d.ensureActionCompleted(srcCluster)
 }
 
+func (d *DRPCInstance) isDiscoveredAppDRPC() bool {
```

Comment:
i would suggest to rename the func name to `isAppTypeDiscovered`. This was more readable for me.

Reply:
isDiscoveredApp() - when calling a method on the DRPC instance we don't need to repeat DRPC.